tests/test-cases/Group6-VIC-Machine/6-16-Config.robot

```diff
@@ -153,6 +153,18 @@ Configure VCH https-proxy
     Should Contain  ${output}  --http-proxy=http://proxy.vmware.com:3128
     Should Not Contain  ${output}  --https-proxy
 
+    ${output}=  Run  bin/vic-machine-linux configure --name=%{VCH-NAME} --target=%{TEST_URL}%{TEST_DATACENTER} --thumbprint=%{TEST_THUMBPRINT} --user=%{TEST_USERNAME} --password=%{TEST_PASSWORD} --timeout %{TEST_TIMEOUT} --http-proxy https://proxy.vmware.com:3128
```

Comment:
As far as I understand, these cases only configure proxy settings but not use them, so any correct proxy urls will be OK.

Reply:
You're correct. Which suggests we should have an additional test that makes sure that we actually _use_ the configured proxy value. That's going to be a little more involved as it would require deployment of a proxy (could be a container in a different VCH if we want to deploy as part of the test) and checking proxy logs to ensure that traffic was routed through it.

Please open a follow up issue for the functional test (as opposed to this configuration test) if that's too involved for this PR.

Reply:
According to #8144, proxy settings are used as environment variables, so that is the very common usage of http proxies.